src/app/shared/api-helper.ts

```diff
@@ -82,7 +82,17 @@ export class ApiHelper {
   static dateRangeFilter(min: string, max: string): string[] {
     return this.rangeFilter(min, this.shiftToDayEnd(max));
   }
-
+  /**
+   * Shift the max date to the end of the day and returns it with the min as range,
+   * suitable for posting a date period on the API
+   */
+  static dateRange(min: string, max: string): DatePeriod {
```

Comment:
Minor suggestion: As the returned type is DatePeriod, why is the method dateRange instead of datePeriod?